### PR TITLE
Fixed backend webpack config for ios

### PR DESCRIPTION
**Pull request by khanaffan**
`build:ios-backend` was failing due to binary dependency recently added. The webconfig has been updated to excluded it.

### CODE REVIEW

**Review by markschlosseratbentley — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

File: test-apps/display-test-app/src/frontend/App.ts [hunk 1]
```diff
@@ -176,6 +176,13 @@ export class DisplayTestApp {
       localhostIpcApp: {
         socketPort: 3002,
       },
+      nativeApp: {
+        authConfig: {
+          clientId: "imodeljs-electron-test",
+          redirectUri: "imodeljs://app/signin-callback",
+          scope: "openid email profile organization imodelhub context-registry-service:read-only reality-data:read product-settings-service projectwise-share urlps-third-party imodel-extension-service-api imodeljs-router offline_access",
+        },
+      }
```
Comment (kabentley) [inline test-apps/display-test-app/src/frontend/App.ts:185]: all changes to this file should be removed.

**Review by kabentley — CHANGES_REQUESTED** [review]

File: test-apps/display-test-app/src/frontend/App.ts (hunk 1 above)
Comment (kabentley) [reply test-apps/display-test-app/src/frontend/App.ts:185]: this should be done on the backend.

**Review by kabentley — CHANGES_REQUESTED** [review]

File: test-apps/display-test-app/src/frontend/App.ts (hunk 1 above)
Comment (khanaffan) [reply test-apps/display-test-app/src/frontend/App.ts]: it works

File: test-apps/display-test-app/src/frontend/App.ts [hunk 2]
```diff
@@ -167,12 +167,6 @@ export class DisplayTestApp {
           uriPrefix: configuration.customOrchestratorUri || "http://localhost:3001",
           info: { title: "DisplayTestApp", version: "v1.0" },
         },
-        authConfig: {
```
Comment (kabentley) [inline test-apps/display-test-app/src/frontend/App.ts:170]: this is still needed for web case. Please remove this change

**Review by kabentley — CHANGES_REQUESTED** [review]

File: test-apps/display-test-app/src/frontend/App.ts (hunk 2 above)
Comment (khanaffan) [reply test-apps/display-test-app/src/frontend/App.ts]: My mistake.

**Review by kabentley — APPROVED** [review]

**Pull request merged**